src/components/DonutChart.js

```diff
@@ -38,6 +38,8 @@ class DonutChart extends React.Component {
     showBaseArc: PropTypes.bool,
     showDataLabel: PropTypes.bool,
     theme: themeShape,
+    toggleDataLabelOnHover: PropTypes.bool,
```

Comment:
this name makes it look like a function or similar to me. I prefer booleans to start with the prefix is, are, should or similar. In this case I think something like "shouldTriggerSliceHoverEvents" or "shouldToggleDataLabelOnHover" would be better options.

Reply:
I'll use shouldToggleDataLabelOnHover. Thanks, this was a difficult thing to try and name